ui/src/stores/ForeignStore.js

```diff
@@ -168,9 +169,9 @@ class ForeignStore {
     setInterval(() => {
       this.getBlockNumber()
       this.getEvents()
```

Comment:
For which scenario are data/actions produced getEvents() used? Is it worth to reduce amount of requests to frame the getting events only for particular cases?